HipercowApi/Tools/MetricsRegistry.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Imperial College London. All rights reserved.
+
+using System.Diagnostics.CodeAnalysis;
+using Prometheus;
+
+/// <summary>
+/// Static registry of Hipercow metrics.
+/// </summary>
+[ExcludeFromCodeCoverage]
```

Comment:
Skipping these from code coverage at the moment - I don't quite know how to test that - perhaps integration tests later.